guides/security/device_profiles.md

```diff
@@ -1,8 +1,10 @@
-# Admin profiles and standard user profiles on devices
+# Device profiles and standard builds
 
-As part of our commitment to secure working practices, we install separate admin and standard user profiles on laptops. The reason this restriction is in place is to prevent the accidental downloading and installation of malicious software on company devices, to close an attack vector, and to protect both Made Tech and our customers.
+Made Tech uses standard builds for the devices issued to team members to facilitate support and maintenance, and enforce company standards in security. Standard builds are not used where the IMS management team and/or leadership have agreed there is a clear case for not doing so, as well as when the application or purpose of platform requires alternative or special configurations.
```

Comment:
If we are using acronyms like "IMS team" we should link to an explanation of these terms as they aren't plain english. The audience of our Handbook includes potential employees and customers, as well as current staff.

Reply:
The README of the IMS Section includes a definition of IMS in the first paragraph. 

Do we need to repeat this definition in every file?